static/include/micro-create-your-own.md

```diff
@@ -3,5 +3,5 @@ If none of the existing models fit your use case, you can create a {{< glossary_
 
 `viam-micro-server` works differently from the RDK, so creating modular resources for it is different.
 Refer to the [Micro-RDK Module Template on GitHub](https://github.com/viamrobotics/micro-rdk/tree/main/templates/module) for information on how to create custom resources for your `viam-micro-server` machine.
-You will need to [recompile and flash your ESP32 yourself](/operate/get-started/setup/) instead of using Viam's prebuilt binary and installer.
+You will need to [recompile and flash your ESP32 yourself](/operate/get-started/other-hardware/micro-module/) instead of using Viam's prebuilt binary and installer.
```

Comment:
```suggestion
You will need to [build custom firmware and flash your ESP32 yourself](/operate/get-started/other-hardware/micro-module/) instead of using Viam's prebuilt binary and installer.
```